server/util/capabilities/capabilities.go

```diff
@@ -57,3 +57,19 @@ func IsGranted(ctx context.Context, env environment.Env, cap akpb.ApiKey_Capabil
 	}
 	return user.HasCapability(cap), nil
 }
+
+func GetFromContext(ctx context.Context, env environment.Env) ([]akpb.ApiKey_Capability, error) {
```

Comment:
I think a better name might be `ForAuthenticatedUser` (so the callsite reads `capablities.ForAuthenticatedUser`) since it might be a bit unclear how we are figuring out the capabilities from the context.

Reply:
Ack - done